HelpMyStreetFE/HelpMyStreetFE/Services/AddressService.cs

```diff
@@ -100,9 +97,23 @@ public async Task<IEnumerable<LocationWithDistance>> GetLocationDetailsForUser(U
         {
             if (user.PostalCode != null)
             {
-                return await _memDistCache_LocationDistanceList.GetCachedDataAsync(async (cancellationToken) => {
+                return await _memDistCache_LocationDistanceList.GetCachedDataAsync(async (cancellationToken) => {    
+                //check if user is member of groupid =-32 if yes set to 2000d else 20d 
+                //this problem will go away when combining requests and shifts
+                int defaultShiftRadius = 20;
+                var groups = await _groupRepository.GetUserGroups(user.ID);
```

Comment:
Better to use the function in the GroupMemberService rather than the one in the GroupRepository, as that will have caching built-in.